FunHouse_Arduino_Demos/analogread/analogread.ino

```diff
@@ -0,0 +1,12 @@
+void setup() {
+  Serial.begin(115200);
+}
+void loop() {
+  // put your main code here, to run repeatedly:
+  uint32_t vol;

```

Comment:
what is 'vol'?

Reply:
Not sure. I think volume? Maybe I should remove this one since it doesn't seem quite as useful as the others.

Reply:
ok remove